melodic/distribution.yaml

```diff
@@ -1556,6 +1556,17 @@ repositories:
       url: https://github.com/fetchrobotics/fetch_tools.git
       version: melodic-devel
     status: maintained
+  fiducials:
+    release:
```

Comment:
Do you want to add a source, doc, and status, similar to what is done in Kinetic?